### PR TITLE
fix: #RDV-29 make feedback css

### PR DESCRIPTION
**Pull request by niko-amosse**
## Issue ticket number and link

[RDV-29](https://jira.support-ent.fr/browse/RDV-29)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)
- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/containers/BookAppointmentGridInfos/index.tsx [hunk 1]
```diff
@@ -44,6 +47,8 @@ export const BookAppointmentGridInfos: FC<BookAppointmentGridInfosProps> = ({
 
   const { duration, videoCallLink, place, publicComment } = gridInfos || {};
 
+  console.log(publicComment);
```
Comment (niko-amosse) [inline frontend/src/containers/BookAppointmentGridInfos/index.tsx:50]: j'ai vu

**Review by Mattox0 — APPROVED** [review]

File: frontend/src/containers/BookAppointmentGridInfos/index.tsx [hunk 2]
```diff
@@ -30,8 +30,11 @@ import {
 } from "./style";
 import { BookAppointmentGridInfosProps } from "./types";
 import { NoAvatar } from "~/components/SVG/NoAvatar";
+import { DURATION_VALUES } from "~/core/constants";
+import { DURATION } from "~/core/enums";
 import { useBookAppointmentModal } from "~/providers/BookAppointmentModalProvider";
 import { GREY } from "~/styles/color.constants";
+import { elipsisWithWrapStyle } from "~/styles/textStyles";
```
Comment (alicedraillard) [inline frontend/src/containers/BookAppointmentGridInfos/index.tsx:37]: ellipsis*

File: frontend/src/styles/textStyles.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const elipsisWithWrapStyle = {
```
Comment (alicedraillard) [inline frontend/src/styles/textStyles.ts:1]: idem ellipsis*

File: frontend/src/styles/textStyles.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  overflow: "hidden",
+  textOverflow: "ellipsis",
+  whiteSpace: "pre-line",
+};
+
+export const elipsisWithNoWrapStyle = {
```
Comment (alicedraillard) [inline frontend/src/styles/textStyles.ts:7]: idem ellipsis*

File: frontend/src/core/constants.ts [hunk 5]
```diff
@@ -6,19 +6,19 @@ export const MAX_STRING_LENGTH = 250;
 
 export const DURATION_VALUES = {
   [DURATION.FIFTEEN_MINUTES]: {
-    displayValue: "15 min",
+    displayValue: "15mn",
```
Comment (alicedraillard) [inline frontend/src/core/constants.ts:9]: pourquoi 'mn' et pas 'min' ? min est le terme officiel

**Review by alicedraillard — APPROVED** [review]

**Pull request merged**